benefit-finder/cypress/fixtures/benefits-eligibility.json

```diff
@@ -0,0 +1,114 @@
+{
+    "scenario_1_covid": {
+        "en": {
+            "param": {
+                "applicant_date_of_birth_month": 4,
+                "applicant_date_of_birth_day": 5,
+                "applicant_date_of_birth_year": 1960,
+                "applicant_relationship_to_the_deceased": "Spouse",
+                "applicant_marital_status": "Widowed",
+                "applicant_citizen_status": "Yes",
+                "applicant_care_for_child": "Yes",
+                "applicant_paid_funeral_expenses": "Yes",
+                "deceased_date_of_death_month": 2,
+                "deceased_date_of_death_day": 3,
+                "deceased_date_of_death_year": 2022,
+                "deceased_death_location_is_US": "Yes",
+                "deceased_paid_into_SS": "Yes",
+                "deceased_public_safety_officer": "No",
+                "deceased_miner": "No",
+                "deceased_american_indian": "No",
+                "deceased_died_of_COVID": "Yes",
+                "deceased_served_in_active_military": "No"
+            },
+            "results": {
+                "eligible": {
+                    "length": 3,
+                    "eligible_benefits": [
+                        "COVID-19 funeral assistance",
+                        "Survivors benefits for mothers/fathers",
+                        "Survivors benefits for spouse"
+                    ]
+                },
+                "total_benefits": 30
+            }
+        }
+    },
+    "scenario_2_veteran": {
+        "en": {
+            "param": {
```

Comment:
Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-ebf47abf1eaef31294ffdbbbc744fb0a6bf48dc3bc09af80080b41b5880d33f7R84-R101